src/main/kotlin/edu/carroll/polyominoes/service/account/RegisterService.kt

```diff
@@ -0,0 +1,31 @@
+package edu.carroll.polyominoes.service.account
+
+interface RegisterService {
+    /**
+     * Given a username, email, and rawPassword, creates a new account in the database.
```

Comment:
I wouldn't necessarily state 'database', as it would be possible this was stored in something else. But, I'm picking nits here.